### DISCUSSION TITLE
tm-modal template attribute

### DISCUSSION THREAD

**Discussion by amreus** [TiddlyWiki/TiddlyWiki5#6938] [Ideas]
so a `tm-modal` message can open a tiddler in a modal dialog. Can we get a `template` attribute to format the tiddler before display? Similar to the template attribute for the `tm-open-window` message.

**Comment by Jermolene** [comment]
Hi @amreus the same result can be achieved by treating the template as the target of the modal, passing the desired currentTiddler variable as a parameter. For example, to render the template "$:/_tw5.com-card-template" with the currentTiddler "HelloThere":

```
<$action-sendmessage $message="tm-modal" $param="$:/_tw5.com-card-template" currentTiddler="HelloThere"/>
```
